core/services/chainlink/application.go

```diff
@@ -351,6 +350,11 @@ func (app *ChainlinkApplication) Start() error {
 		}
 	}
 
+	// EthBroadcaster needs to be started after GasUpdater (in subservices)
+	if err := app.EthBroadcaster.Start(); err != nil {
```

Comment:
Could this requirement be enforced through code rather than a comment? e.g: Maybe the `GasUpdater#New` function requires you to pass in a `EthBroadcaster` that is `Started() == true` ?